src/PrintOrder/ComponentToPrint/ComponentToPrint.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import { FormattedMessage, IntlProvider, useIntl } from 'react-intl';
+
+import {
+  Col,
+  Grid,
+  KeyValue,
+  Label,
+  NoValue,
+  Row,
+} from '@folio/stripes/components';
+import {
+  AmountWithCurrencyField,
+  FolioFormattedTime,
+  ORDER_STATUS_LABEL,
+} from '@folio/stripes-acq-components';
+
+import {
+  DISCOUNT_TYPE,
+} from '../../components/POLine/const';
+import { LINE_FIELDS_MAP, LINE_FIELDS_LABELS } from './constants';
+import css from './ComponentToPrint.css';
+
+function PrintValue({ path, source, exportedLine }) {
+  const value = get(exportedLine, path, null) || get(source, path, null);
+  const currency = get(source, LINE_FIELDS_MAP.currency);
+
+  switch (path) {
+    case LINE_FIELDS_MAP.listUnitPrice:
+    case LINE_FIELDS_MAP.listUnitPriceElectronic:
+    case LINE_FIELDS_MAP.additionalCost:
+    case LINE_FIELDS_MAP.poLineEstimatedPrice:
+      return (
+        <AmountWithCurrencyField
+          currency={currency}
+          amount={value}
+        />
+      );
+    case LINE_FIELDS_MAP.discount:
+      if (!value) return <NoValue />;
+
+      return get(source, 'cost.discountType') === DISCOUNT_TYPE.percentage
+        ? `${value}%`
+        : (
+          <AmountWithCurrencyField
+            currency={currency}
+            amount={value}
+          />
+        );
+    default:
+      if (value === true) {
+        return <FormattedMessage id="ui-orders.filter.true" />;
+      } else if (value === false) {
+        return <FormattedMessage id="ui-orders.filter.false" />;
+      }
+
+      return value ?? <NoValue />;
+  }
+}
+
+const ComponentToPrint = ({ dataSource = {} }) => {
+  const intl = useIntl();
+
+  return (
+    <IntlProvider
+      defaultLocale="en"
+      messages={intl.messages}
+      timeZone={intl.timeZone}
+      currency={intl.currency}
+    >
+      <Grid>
+        <Row>
+          <Col xs={6}>
+            <Label>
+              <FormattedMessage id="ui-orders.print.po" />
+            </Label>
+          </Col>
+          <Col xs={6}>
+            <div>
+              <FormattedMessage id="ui-orders.orderSummary.workflowStatus" />:&nbsp;
+              {dataSource.workflowStatus ? ORDER_STATUS_LABEL[dataSource.workflowStatus] : <NoValue />}
+            </div>
+          </Col>
+        </Row>
+        <Row>
+          <Col xs={6}>
+            <div>
+              <FormattedMessage id="ui-orders.orderSummary.closingReason" />:&nbsp;
```

Comment:
don't we have keyvalue but for horizontal?

Reply:
I think no, we don't have such one. Also I'm not sure that we need to make labels in bold.